x-pack/plugins/enterprise_search/public/applications/workplace_search/views/role_mappings/role_mappings.tsx

```diff
@@ -57,6 +74,8 @@ export const RoleMappings: React.FC = () => {
       pageChrome={[ROLE_MAPPINGS_TITLE]}
       pageHeader={{ pageTitle: ROLE_MAPPINGS_TITLE }}
       isLoading={dataLoading}
+      isEmptyState={roleMappings.length < 1}
```

Comment:
[non-blocking] I think this could have used some additional test updates, maybe something like:

```
it('shows an empty state when there are no role mappings', () => {
  // Assert that isEmptyState was passed as true, and maybe that emptyState contains a properly configured RolesEmptyPrompt. Like assert that enableRoleBasedAccess was pulled from state and passed to onEnable.
})
```

Reply:
Added to Google doc for tech debt after 7.14 FF